derive/src/lib.rs

```diff
@@ -39,25 +40,39 @@ impl FieldAttrs {
         for attr in attrs.iter() {
             match attr.parse_meta()? {
                 Meta::List(MetaList { path, nested, .. }) => {
-                    if path.get_ident().unwrap() == "jayson" {
+                    if matches!(
+                        path.get_ident().unwrap().to_string().as_str(),
+                        "jayson" | "serde"
```

Comment:
why do we also match `serde` here? We don't want to interfere with potential serde implementations

Reply:
The idea is to temporarily use this only during Jayson's development and the transition away from serde_json. 
It is a convenient (but not strictly necessary) way to check that:

1. every serde attribute we use is also supported by Jayson, because it will return an error otherwise. Of course if we want to ignore some serde attributes, that is possible as well.
2. The common attributes between serde and Jayson behave in the same way (as checked by the unit tests here).

It doesn't interfere with serde implementations in any way so long as we don't add jayson-specific attributes under the serde attribute name.

Reply:
Actually, never mind. Since we'll need other, more complex attributes later on, I have decided to rewrite the attribute parser. In that new parser, we will only accept `jayson`. I will make a PR for it tomorrow.